startvscode.sh

```diff
@@ -20,7 +20,7 @@ if [ ! -f "$DOTNET_ROOT/dotnet" ]; then
     exit 1
 fi
 
-if [ $1 = "" ]; then
+if [[ $1 == "" ]]; then
```

Comment:
To more correctly handle a missing positional parameter and use the usual "is it empty❔" operator&hellip;
```suggestion
if [[ -z "${1:-}" ]]; then
```

Reply:
Sorry for late response. 
Thanks for your advice and I think it should be equal at most cases to my knowledge.
Thanks again for review and merging this PR.